pset-functions.js

```diff
@@ -75,6 +104,44 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+function scrabbleScore(word) {
+  let score = 0
+
+  // make sure it's a string
+  // testing whether a string is an actual word
+  // (or an accepted Scrabble word)
+  // is much harder
+  if (typeof word === "string") {
+    word = word.toLowerCase()
+  } else {
+    return "scrabbleScore() requires a string parameter."
+  }
+
+  // going through character and giving them points values
+  for (i = 0; i < word.length; i++) {
+    if (word[i] === 'q' || word[i] === 'z') {
+      score += 10
+    } else if (word[i] === 'j' || word[i] === 'x') {
+      score += 8
+    } else if (word[i] === 'k') {
+      score += 5
+    } else if (word[i] === 'f' || word[i] === 'h' || word[i] === 'v' || word[i] === 'w' || word[i] === 'y') {
+      score += 4
+    } else if (word[i] === 'b' || word[i] === 'c' || word[i] === 'm' || word[i] === 'p') {
+      score += 3
+    } else if (word[i] === 'd' || word[i] === 'g') {
+      score += 2
+    } else if (word[i] === 'a' || word[i] === 'e' || word[i] === 'i' || word[i] === 'o' || word[i] === 'u' || word[i] === 'l' || word[i] === 'n' || word[i] === 'r' || word[i] === 's' || word[i] === 't') {
```

Comment:
Nate, another great use of if/else statements in combination with the "or" operator to test your conditions. It's clear you are also keeping edge cases in mind (very well done). Another potential solution to using several "or" operands takes advantage of JS' built in method, .includes. For an example of this, please take a look [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L98)